conf/output/test-hebis-to-lobid-output-10.json

```diff
@@ -0,0 +1,122 @@
+{
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "id" : "http://lobid.org/resources/090358163#!",
+  "type" : [ "BibliographicResource", "Book" ],
+  "medium" : [ {
+    "label" : "Print",
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010"
+  } ],
+  "title" : "Mainz transparent",
+  "almaMmsId" : "090358163",
+  "isbn" : [ "3874394883", "9783874394888" ],
+  "oclcNumber" : [ "76115483" ],
+  "dnbId" : "958853371",
+  "otherTitleInformation" : [ "2000 Jahre einer Stadt ; [Stadtbegleiter]" ],
+  "publication" : [ {
+    "startDate" : "2000",
+    "type" : [ "PublicationEvent" ],
+    "location" : [ "Mainz" ],
+    "publishedBy" : [ "Schmidt" ]
+  } ],
+  "sameAs" : [ {
+    "id" : "http://worldcat.org/oclc/76115483",
+    "label" : "OCLC Ressource"
+  }, {
+    "id" : "https://d-nb.info/958853371",
+    "label" : "DNB-Ressource"
+  } ],
+  "language" : [ {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/ger",
+    "label" : "Deutsch"
+  } ],
+  "extent" : "127 S. : zahlr. Ill., Kt.",
+  "natureOfContent" : [ {
+    "label" : "Führer",
+    "id" : "https://d-nb.info/gnd/4155569-7"
+  } ],
+  "subject" : [ {
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "Gemeinsame Normdatei (GND)",
+      "id" : "https://d-nb.info/gnd/7749153-1"
+    },
+    "label" : "Mainz",
+    "id" : "https://d-nb.info/gnd/4037124-4",
+    "gndIdentifier" : "4037124-4"
+  }, {
+    "notation" : "61",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "Systematik der DNB (bis 2003)",
+      "id" : "https://bartoc.org/en/node/18497"
+    }
+  }, {
```

Comment:
This `subject` entry is to spars wrt the lobid-resources JSON schema, see https://github.com/hbz/rpb/pull/105#issuecomment-2689971143